src/main/webapp/WEB-INF/tags/statusMessage.tag

```diff
@@ -11,7 +11,7 @@
         <div id="statusMessagesToUser">
             <c:forEach items="${statusMessagesToUser}" var="statusMessage">
                 <div class="overflow-auto alert alert-${statusMessage.color} statusMessage">
-                    ${statusMessage.text}
+                        ${statusMessage.text}
```

Comment:
Unrelated whitespace change. Once this is removed I think the entire PR should collapse to a single changed file (`teammatesCommon.css`).